README.md

```diff
@@ -1,3 +1,202 @@
-# UNDER DEVELOPMENT
-# r16n
-I18n for Redux
+<img src="https://cdn.rawgit.com/razodeh/r16n/ab1ec653/logo/logo-title.png" height="50px" alt="R16N Logo" /> 
+
+# Reduxtionalization (Redux I18N) [![Build Status](https://travis-ci.org/razodeh/r16n.svg?branch=master)](https://travis-ci.org/razodeh/r16n) 
+
```

Comment:
Let's add an About section here, so people know exactly what this library is about once they land on the page.